### ISSUE TITLE
specify kingdom?

### ISSUE THREAD

**Issue by ctc3d** [EDIorg/taxonomyCleanr#36]
The project I'm working on involves only tree inventories.  A confused data collector somehow entered 'Phanus' when they meant 'Rhamnus' (buckthorn), but since Phanus is a perfectly good butterfly it passed right through and was only caught in a later stage.  Specifying kingdoms would help prevent this kind of false match.  I guess in `resolve_sci_taxa()`?  Something like `kingdom=c('Plantae', 'Archaea')`?

If you wanted to get fancy you could generalize this to be more specific, to limit the space to any taxa -- something like `top=c('Mixiomycetes', 'Cnidaria', 'Yersinia')`.  I see implementation troubles with this... but anyway it's a thought.

**Comment by clnsmth** [comment]
Good idea @ctc3d. Another approach (long ago suggested by @twhiteaker) could be the output of a phylogeny plot to visually identify outliers.
